src/main/scala/com/codacy/analysis/cli/files/FileSystemFileCollector.scala

```diff
@@ -9,14 +10,17 @@ import com.codacy.analysis.cli.clients.api.{FilePath, PathRegex, ProjectConfigur
 import com.codacy.analysis.cli.configuration.CodacyConfigurationFile
 import com.codacy.analysis.cli.tools.Tool
 import com.codacy.api.dtos.{Language, Languages}
+import org.log4s.{Logger, getLogger}
 
 import scala.util.Try
 
-final case class FilesTarget(directory: File, files: Set[Path])
+final case class FilesTarget(directory: File, readableFiles: Set[Path], unreadableFiles: Set[Path])
 
 class FileSystemFileCollector extends FileCollector[Try] {
 
-  // TODO: Check if File will work or if we might need Path to support relative paths
+  private case class CheckedFiles(unreadableFiles: Set[Path], readableFiles: Set[Path])
```

Comment:
Should be consistent with FilesTarget (readable first and unreadable after or vice versa)